File: Example Code/01 Variables, Data Types, Input and Output/SongLyrics2.java
import java.util.Scanner;

public class SongLyrics2
{
	public static void main( String args[] )
	{
        // Using input and concatenation
        Scanner scanner = new Scanner( System.in );
        
        System.out.print( "Enter an activity: " );
        String activity = scanner.nextLine();
        System.out.print( "Enter an inter-personal relationship: " );
        String relationship = scanner.nextLine();
        
        System.out.println( "\n\nSAFETY " + activity.toUpperCase() + " - MEN WITHOUT HATS" );
        System.out.println( "We can " + activity + " if we want to." );
        System.out.println( "We can leave your " + relationship + "s behind." );
        System.out.println( "'Cuz your " + relationship + "s don't " + activity + "," );
        System.out.println( "and if they don't " + activity + "," );
        System.out.println( "well they're no " + relationship + "s of mine." );
    }
}

File: Example Code/01 Variables, Data Types, Input and Output/SongLyrics2.cpp
#include <iostream>
#include <string>
using namespace std;

int main()
{
    // Get new lyrics
    cout << "Enter an activity: ";
    string activity;
    cin >> activity;
    
    cout << "Enter an inter-personal relationship: ";
    string relationship;
    cin >> relationship;
    
    // Just display song lyrics
    cout << "SAFETY " << activity << " - MEN WITHOUT HATS" << endl;
    cout << "We can " << activity << " if we want to."  << endl;
    cout << "We can leave your " << relationship << "s behind."  << endl;
    cout << "'Cuz your " << relationship << "s don't " << activity << ","  << endl;
    cout << "and if they don't " << activity << ","  << endl;
    cout << "well they're no " << relationship << "s of mine."  << endl;
    
    return 0;
}
